eng/Version.Details.xml

```diff
@@ -305,13 +305,13 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>0e0e648770e54b12c2fa81a77538ce1a72fca8af</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="5.0.0-beta.20377.2">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="5.0.0-beta.20374.1">
```

Comment:
@Anipik to confirm: When you did the manual Helix SDK upgrade in the preview8 branch, did you use the '.NET Eng - Validation' instead of '.NET Eng - Latest' or otherwise move to the most recent / unvalidated Arcade build❔

Reply:
/ping @Anipik